src/assets/javascript/analytics/ga4/pageViewTracker.js

```diff
@@ -1,54 +1,52 @@
 /* global window document */
 
-window.DI = window.DI || {}
-window.DI.analyticsGa4 = window.DI.analyticsGa4 || {}
+window.DI = window.DI || {};
+window.DI.analyticsGa4 = window.DI.analyticsGa4 || {};
 window.DI.analyticsGa4.trackers = window.DI.analyticsGa4.trackers || {};
 
 (function (trackers) {
-
-  'use strict'
+  "use strict";
 
   let PageViewTracker = {
-
-    init: function() {
+    init: function () {
       const data = {
-        event: 'page_view_ga4',
+        event: "page_view_ga4",
         page_view: {
           language: this.getLanguage(),
           location: this.getLocation(),
-          organisations: '<OT1056>',
-          primary_publishing_organisation: 'government digital service - digital identity',
+          organisations: "<OT1056>",
+          primary_publishing_organisation:
+            "government digital service - digital identity",
           status_code: this.getStatusCode(),
           title: this.getTitle(),
           referrer: this.getReferrer(),
-          taxonomy_level1: 'web cri',
-          taxonomy_level2: 'cic'
-        }
-      }
-      window.DI.core.sendData(data)
+          taxonomy_level1: "web cri",
+          taxonomy_level2: "cic",
+        },
+      };
+      window.DI.core.sendData(data);
     },
 
-    getLanguage: function() {
-      return (window.DI.cookies.getCookie('lng') ?? 'en').toLowerCase()
+    getLanguage: function () {
+      return (window.DI.cookies.getCookie("lng") ?? "en").toLowerCase();
     },
 
-    getStatusCode: function() {
-      return window.DI.httpStatusCode ?? 200
-    },
-    
-    getLocation: function() {
-      return document.location.href?.toLowerCase()
+    getStatusCode: function () {
+      return window.DI.httpStatusCode ?? 200;
     },
-    
-    getTitle: function() {
-      return document.title?.toLowerCase()
+
+    getLocation: function () {
```

Comment:
I'm a little concerned about this, did you do it yourself or was it part of the lint fix?

Reply:
Oh wait actually I think this is just github being weird the `getLocation` hasn't actually been changed 👍